rs/messaging/src/routing/stream_handler/tests.rs

```diff
@@ -1715,16 +1715,17 @@ fn induct_stream_slices_reject_response_from_old_host_subnet_is_accepted() {
                 &mut expected_state,
                 message_in_slice(slices.get(&CANISTER_MIGRATION_SUBNET), 0).clone(),
             );
-            // ...and a stream to `CANISTER_MIGRATION_SUBNET` with all 3 messages accepted.
+            // ...and a stream to `CANISTER_MIGRATION_SUBNET` with 2 message accepted and 1 rejected.
```

Comment:
```suggestion
            // ...and a stream to `CANISTER_MIGRATION_SUBNET` with the 2 responses accepted and the request rejected.
```

Doc comment of the test case also needs updating.